packages/xo-web/src/common/usage/index.js

```diff
@@ -20,12 +27,27 @@ const Usage = ({ total, children }) => {
         (child, index) =>
           child.props.value > limit && cloneElement(child, { total })
       )}
-      <Element others tooltip={_('others')} total={total} value={othersTotal} />
+      <Element
+        others={!tooltipOthers}
+        tooltip={
+          tooltipOthers && type !== undefined && nOthers > 0
+            ? _('tooltipOthers', {
+                nOthers,
+                total: formatSize(othersTotal),
+                type,
+              })
+            : _('others')
+        }
+        total={total}
+        value={othersTotal}
+      />
```

Comment:
This was originally meant to be an easy way of grouping small disks together for a simple linear and flat structure. Now that it's a bit more complex, if this doesn't fit your needs, you don't have to use it.

Reply:
I need it for more information about `Other` in order to be clearer for the user and this behavior required by the issue.